packages/blade/src/components/Dropdown/DropdownOverlay.native.tsx

```diff
@@ -8,42 +8,44 @@ import Box from '~components/Box';
 import { makeSize, metaAttribute, MetaConstants } from '~utils';
 import type { WithComponentId } from '~utils';
 
-const StyledDropdownOverlay = styled(Box)((props) => ({
+const StyledDropdownOverlay = styled(Box)<{ testID: 'dropdown-overlay' }>((props) => ({
   transform: `translateY(${makeSize(props.theme.spacing[3])})`,
 }));
 
-const StyledClosableArea = styled(Pressable)<{ display: 'flex' | 'none' }>((props) => ({
+const StyledCloseableArea = styled(Pressable)<{ display: 'flex' | 'none' }>((props) => ({
   position: 'static',
   height: '100%',
   width: '100%',
   display: props.display,
 }));
 
 /**
- * Overlay for dropdown.
+ * Overlay of dropdown
  *
- * Wrap your ActionList with this this component
+ * Wrap your ActionList within this component
  */
 const DropdownOverlay: WithComponentId<DropdownOverlayProps> = ({ children }): JSX.Element => {
   const { isOpen, setIsOpen } = useDropdown();
 
   return (
     <Box position="relative">
-      <StyledClosableArea
+      <StyledCloseableArea
         display={isOpen ? 'flex' : 'none'}
         onPress={() => {
           setIsOpen(false);
         }}
+        testID="closeable-area"
```

Comment:
possible to use `testID()` function from utils? 

Plus is the data-testid set on web side?

Reply:
nope. On web I can use `getByRole` with a name option on listbox/menu but in the `@testing-library/react-native` version that we're using, it doesn't seem to be supported.

Reply:
Do I use `testID` util if I only want it in react-native? don't see any harm in adding `data-testid` as well just that I wouldn't recommend using that for web tests 🤔

Reply:
> Do I use testID util if I only want it in react-native? don't see any harm in adding data-testid as well just that I wouldn't recommend using that for web tests 🤔

Sure, if we are not doing any testing with the testId let's not add it on the web.